src/common-components/Logistration.jsx

```diff
@@ -12,19 +16,47 @@ import { RegistrationPage } from '../register';
 
 const Logistration = (props) => {
   const { intl, selectedPage } = props;
+  const [institutionLogin, setInstitutionLogin] = useState(false);
+
+  const handleInstitutionLogin = () => {
+    if (selectedPage === LOGIN_PAGE) {
```

Comment:
Why we are sending this event only for the login page? I believe we need to send it for both login and register.

Reply:
Previously, this event tracking was only enabled for the Login screen. I moved this logic from the Login component to Registration. Please let me know if we need to send an event for Registration also

Reply:
Yes, it should be for both login and registration. I guess it was implemented incorrectly in the past..

Reply:
Done